UI/Application.html

```diff
@@ -0,0 +1,166 @@
+					</h3>
+					<div class="section">
+						<div class="section-left">
+							<h6>Amount of candidates</h6>
+							<div class="vote">
+								<button class="vote-up">
+									
+								</button>
+								<span>113</span>
+								<button class="vote-down">
+									
+								</button>
+							</div>
+						</div>
+						<div class="section-right">
+							<div class="application-card-body-content">
+								<span class="application-card-comments">
+									Here is the parties <b>OBJECTIVE</b>
+								</span>
+								<div class="application-card-description">
+									<p>
+										Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+										tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+										quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+										consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+										cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+										proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+									</p>
+								</div>
+							</div>
+						</div>
+					</div>
+					<br>
+					<div class="application-card-comment-area">
+								<form>
+									<div class="application-card-input-section">
+										<textarea 
+											placeholder="Manifesto goes here" 
+											required 
+										></textarea>
+									</div><br>
+									<div class="application-card-button-area">
+										<button>Submit</button>
+									</div>
+								</form>
+							</div>
+					<div class="application-card-footer">
+						<small>The people Democratic Party</small>
+					</div>
+				</div>
+
+ 				<div class="apply-area">
+					 <h2>Top Candidates</h2>
+					<div class="who-applied">
+						<h4>Taiwo Oyindamola</h4>
+						<small>10:30am, 20th Dec 2018</small>
+					</div>
+
+				</div>
+				<div class="apply-area">
+					<div class="who-applied">
+						<h4>Segun Obey</h4>
+						<small>2:21pm, 31th Dec 2018</small>
+					</div>
+	
+				</div>
+				<div class="apply-area">
+					<div class="who-applied">
+						<h4>Tunde Zack</h4>
+						<small>11:00pm, 1st Dec 2018</small>
+					</div>
+
+				</div>
+			</div>
+		</div>
+		<div class="signup-box" id="personal-details-card">
+			<BR></BR>
+				<form action="./Application.html">
+					<div class="signup-input-section">
+						<input type="text" placeholder="First Name"/>
+					</div>
+					<div class="signup-input-section">
+						<input type="text" placeholder="Last Name"/>
+					</div>
+					<div class="signup-input-section">
+						<input type="email" placeholder="Email"/>
+					</div>
+					<div class="signup-submit">
+						<button id="submitButton" type="submit">Submit</button>
+					</div>
+				</form>
+
+		<footer class="politico-footer">
+			<p>politico. &copy; 2019</p>
+		</footer>
+        <script src="./assets/js/script.js"></script> 
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks